pgmpy/independencies/Independencies.py

```diff
@@ -42,7 +42,7 @@ class Independencies(object):
     get_factorized_product
     """
     def __init__(self, *assertions):
-        self.independencies = []
+        self.independencies = set()
```

Comment:
Let it be a list, maybe if user want to have assertions like `(X_|_ Y,Z | W)` or `(Z,Y_|_X | W) `. reduce function should remove them and also, case like `(X _|_ Y | W)`